testdata/cheats/Env.t.sol

```diff
@@ -901,34 +910,34 @@ contract EnvTest is DSTest {
         );
     }
 
-    function testEnvOrBytes32EmptyArrKey() public {
-        string memory key = "_foundryCheatcodeEnvOrBytes32EmptyArrTestKey";
-        string memory value = "";
-        bytes32[] memory expected = new bytes32[](0);
-        bytes32[] memory defaultValues = new bytes32[](0);
-
-        vm.setEnv(key, value);
-        string memory delimiter = ",";
-        bytes32[] memory output = vm.envOr(key, delimiter, defaultValues);
-        require(
-            keccak256(abi.encodePacked((output))) == keccak256(abi.encodePacked((expected))),
-            "envOrBytes32EmptyArrKey failed"
-        );
-    }
-
-    function testEnvOrBytes32EmptyArrDefault() public {
-        string memory key = "_foundryCheatcodeEnvOrBytes32EmptyArrTestDefault";
-        string memory value = "";
-        bytes32[] memory expected = new bytes32[](0);
-        bytes32[] memory defaultValues = new bytes32[](0);
-
-        string memory delimiter = ",";
-        bytes32[] memory output = vm.envOr(key, delimiter, defaultValues);
-        require(
-            keccak256(abi.encodePacked((output))) == keccak256(abi.encodePacked((expected))),
-            "envOrBytes32EmptyArrDefault failed"
-        );
-    }
+    // function testEnvOrBytes32EmptyArrKey() public {
```

Comment:
Previous implementation of `parse_tokens`/`parse` padded all values to 32 bytes if they weren't the right size. This seems more like a bug rather than a feature, but curious about what y'all think @mattsse / @DaniPopes

Reply:
what is the intended behavior?

Reply:
Previously any value that wasn't exactly 32 bytes was padded—now we're more strict and we don't do that. This could be considered a breaking change, so I'm wondering what we wanna do here. I'd prefer not padding values actually